src/base/bittorrent/session.cpp

```diff
@@ -2282,41 +2290,51 @@ void Session::handleIPFilterError()
     emit ipFilterParsed(true, 0);
 }
 
+#if LIBTORRENT_VERSION_NUM < 10100
 void Session::dispatchAlerts(std::auto_ptr<libt::alert> alertPtr)
 {
     QMutexLocker lock(&m_alertsMutex);
 
-    bool wasEmpty = m_alerts.isEmpty();
+    bool wasEmpty = m_alerts.empty();
 
-    m_alerts.append(alertPtr.release());
+    m_alerts.push_back(alertPtr.release());
```

Comment:
IIRC its better to use `emplace_back` whenever possible. (probably doesn't matter much in this case)